backend/app/routes/broadcast/index.js

```diff
@@ -0,0 +1,10 @@
+const router = require('express').Router({ mergeParams: true });
+const validationMiddleware = require('../../../helpers/middlewares/validation');
+const { authMiddleware } = require('../../../helpers/middlewares/auth');
+
+const { addBroadcastValidation } = require('./@validationSchema');
+const addBroadcast = require('./addBroadcast');
+
+router.post('/add', validationMiddleware(addBroadcastValidation), authMiddleware, addBroadcast);
```

Comment:
it should be just `/broadcast` and not `/broadcast/add`

Reply:
I thought /broadcast will be used for getting broadcasts.

Reply:
Yes but that will be a get request

Reply:
ok i will change it